### PR TITLE
New logo and recipe images

### LINKED ISSUE #92: Find a new logo
I don't think it's appropriate for us to be using the logo from another project. [Flaticon](http://www.flaticon.com/icon-packs/cook) has a bunch of potential options worth looking at:

http://www.flaticon.com/packs/kitchen-8
http://www.flaticon.com/packs/kitchenware-2

### PR DESCRIPTION
**Pull request by hermzz**
Also makes the image in the recipe boxes clickable.

Fixes #92

### CODE REVIEW

File: frontend/modules/base/components/Footer.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ class Footer extends React.Component{
                 link: <a href="https://github.com/RyanNoelk/OpenEats">OpenEats</a>
               }}
             />
-          </p>
+            &nbsp;- 
+            Icons made by <a href="http://www.flaticon.com/authors/nikita-golubev" title="Nikita Golubev">Nikita Golubev</a> from <a href="http://www.flaticon.com" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></p>
```
Comment (RyanNoelk) [inline frontend/modules/base/components/Footer.js]: Translations?

**Review by RyanNoelk — COMMENTED** [review]
Minor nit about trans, otherwise LGTM. I can provide the German translations in another PR. My girlfriend (aka German translator) is away ATM.

File: frontend/modules/base/components/Footer.js (hunk 1 above)
Comment (hermzz) [reply frontend/modules/base/components/Footer.js]: Damn! And right after another translated chunk too!

**Review by RyanNoelk — APPROVED** [review]
LGTM

**Pull request merged**